cmake/faster_build.cmake

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include(CMakeParseArguments)
+
+function(ie_faster_build TARGET_NAME)
+    if(NOT ENABLE_FASTER_BUILD)
+        return()
+    endif()
+
+    cmake_parse_arguments(IE_FASTER_BUILD "UNITY" "" "PCH" ${ARGN})
+
+    if(IE_FASTER_BUILD_UNITY)
+        set_target_properties(${TARGET_NAME}
+            PROPERTIES
+                UNITY_BUILD ON
+        )
+    endif()
+
+    if(IE_FASTER_BUILD_PCH)
+        target_precompile_headers(${TARGET_NAME}
```

Comment:
should not we use `PRIVATE` according to https://cmake.org/cmake/help/git-stage/command/target_precompile_headers.html ?

Reply:
It is used in per-target `ie_faster_build` invocations. The function just forwards the arguments to `target_precompile_headers`.